cayleypy/graphs_lib_test.py

```diff
@@ -278,7 +322,14 @@ def test_three_cycles():
 def test_three_cycles_0ij():
     graph = PermutationGroups.three_cycles_0ij(4)
     assert graph.n_generators == 6
-    expected_generators = [[1, 2, 0, 3], [1, 3, 2, 0], [2, 0, 1, 3], [2, 1, 3, 0], [3, 0, 2, 1], [3, 1, 0, 2]]
+    expected_generators = [
```

Comment:
Again, please don't touch unrelated code in your PR.

It seems like your IDE has different column limit, change it to 120.